core/templates/dev/head/services/AssetsBackendApiService.ts

```diff
@@ -91,10 +91,26 @@ oppia.factory('AssetsBackendApiService', [
                          window.MozBlobBuilder ||
                          window.MSBlobBuilder;
           if (exception.name === 'TypeError' && window.BlobBuilder) {
-            var blobBuilder = new BlobBuilder();
-            blobBuilder.append(data);
-            assetBlob = blobBuilder.getBlob(assetType.concat('/*'));
+            try {
+              var blobBuilder = new BlobBuilder();
+              blobBuilder.append(data);
+              assetBlob = blobBuilder.getBlob(assetType.concat('/*'));
+            } catch (e) {
+              var additionalInfo = (
```

Comment:
Would it be possible to make the two error messages a bit different so that it's possible to see where in the code they come from? (The stack trace line numbers may be unreliable due to minification.)

Reply:
Hi, The starting line of both is different. One is blob construction error log and other is blobbuilder, so won't we know from where they come?

Reply:
Ah yes, I missed that, sorry. LGTM then. Thanks!